src/adapters/interlay.spec.ts

```diff
@@ -19,6 +91,31 @@ describe.skip("interlay-adapter should work", () => {
     return firstValueFrom(provider.connectFromChain(chains, undefined));
   }
 
+  test("connect kintsugi to do xcm", async () => {
+    const fromChains = ["kintsugi", "heiko"] as ChainName[];
+
+    await connect(fromChains);
+
+    const kintsugi = new KintsugiAdapter();
+    const heiko = new HeikoAdapter();
+
+    await kintsugi.setApi(provider.getApi(fromChains[0]));
+    await heiko.setApi(provider.getApi(fromChains[1]));
+
+    const bridge = new Bridge({
+      adapters: [kintsugi, heiko],
+    });
+
+    expect(
+      bridge.router.getDestinationChains({
+        from: chains.kintsugi,
+        token: "KBTC",
+      }).length
+    ).toEqual(1);
+
+    await runMyTestSuite(testAccount, bridge, "kintsugi", "heiko", "KBTC");
```

Comment:
I think this only tests one direction, right? Shouldn't we do both?

Reply:
> I think this only tests one direction, right? Shouldn't we do both?

Correct. This test file is for testing outgoing XCM from interlay/kintsugi.
We could add a parallel/heiko spec file for the other direction, too. 

I personally am happy enough if I can see the return direction's extrinsics generated in `bridge.spec.ts` which are then used for tests on chopsticks.
Having said that, I can see value in adding parallel specs, too.

Reply:
Added the reverse direction in the latest [commit](https://github.com/interlay/bridge/pull/50/commits/fb918ff3f6884703fd25112eecf72a1a21c41758)